extension/background/background.ts

```diff
@@ -0,0 +1,2 @@
+// tslint:disable-next-line:no-console
+export default console.log('Hello world!!!');
```

Comment:
Console log?

Reply:
:)) for now, it's just a test to see that the background script is loaded and working.